exporter/terra/terra_exporter.py

```diff
@@ -0,0 +1,43 @@
+from ingest.api.ingestapi import IngestApi
+from exporter.metadata import MetadataResource, MetadataService, DataFile
+from exporter.graph.graph_crawler import GraphCrawler
+from exporter.terra.dcp_staging_client import DcpStagingClient
+from typing import Iterable
+
+
+class TerraExporter:
+    def __init__(self,
+                 ingest_client: IngestApi,
+                 metadata_service: MetadataService,
+                 graph_crawler: GraphCrawler,
+                 dcp_staging_client: DcpStagingClient):
+        self.ingest_client = ingest_client
+        self.metadata_service = metadata_service
+        self.graph_crawler = graph_crawler
+        self.dcp_staging_client = dcp_staging_client
+
+    def export(self, process_uuid, submission_uuid, experiment_uuid, experiment_version):
```

Comment:
what's the experiment_uuid / version? is it the same as bundle uuid + version?

Reply:
Yep it will be the UUID+version included in the new-assay messages generated by core, currently named "bundleUuid" I believe. Since we're not using bundles anymore we should consider renaming these. Downstream developers like the term "experiment" or "experiment graph", I also think it's a good name for assays/analysis "bundles"